app/src/main/java/org/schabi/newpipe/ktx/OffsetDateTime.kt

```diff
@@ -1,29 +0,0 @@
-package org.schabi.newpipe.ktx
-
-import java.time.OffsetDateTime
-import java.time.ZoneOffset
-import java.time.temporal.ChronoField
-import java.util.Calendar
-import java.util.Date
-import java.util.GregorianCalendar
-import java.util.TimeZone
-
-// This method is a modified version of GregorianCalendar.from(ZonedDateTime).
-// Math.addExact() and Math.multiplyExact() are desugared even though lint displays a warning.
-@SuppressWarnings("NewApi")
-fun OffsetDateTime.toCalendar(): Calendar {
```

Comment:
The bug that you tried to fix with this was resolved by prettytime?

Reply:
I added a `formatUnrounded` method to `PrettyTime` that accepts an `OffsetDateTime`.